software/ChromeAndPuppeteer.md

```diff
@@ -0,0 +1,93 @@
+# (Headless) Chrome and Puppeteer on Graviton
```

Comment:
We need a toplevel page and a link on the main page to get people to this section, otherwise its effectively hidden.

Reply:
Move this to the top level directory?

Reply:
Readme file has it.